source/javascripts/controllers/_MainController.js.erb

```diff
@@ -59,6 +61,21 @@
 				viewModel.menuSelected(initialMenu);
 			}
 
+			function checkRepoYmlSplashModalVisibility () {
+				$q.all([requestService.getCurrentUserBetaTags(), appService.getPipelineConfig()])
+					.then(function(result) {
+						var betaTags = result[0];
+						
+						var featureAvailable = betaTags.indexOf('yml-from-repo') > -1;
```

Comment:
```suggestion
						var featureAvailable = _.contains(betaTags, 'yml-from-repo');
```

Reply:
it does the same in the background, i don't see a benefit of using a lodash method here

Reply:
Easier readability, I think it's worth it.